Gravity/Gravity/DAL/RSAPI/RsapiDao.Get.cs

```diff
@@ -118,13 +120,15 @@ internal void PopulateChildrenRecursively<T>(BaseDto baseDto, RDO objectRdo, Obj
 
 		private object GetChildObjectRecursively(BaseDto baseDto, RDO objectRdo, ObjectFieldsDepthLevel depthLevel, PropertyInfo property)
 		{
+			var fieldType = property.GetCustomAttribute<RelativityObjectFieldAttribute>()?.FieldType;
```

Comment:
Instead of this, try and find the `RelativityObjectFieldAttribute` and wrap the next two blocks in an "if not null" statement. We shouldn't be looking up the attribute twice in a row (reflection is expensive), and it better communicates intent (the FieldGuid can never be null when the field type is set).

Reply:
Done